wallet-webapps-common/src/main/webapp/vue-app/components/ProfileChip.vue

```diff
@@ -40,7 +40,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     <template v-else-if="disabledInRewardPool">
       {{ displayName }} <span class="red--text">({{ $t('exoplatform.wallet.label.disabledPool') }})</span>
     </template>
-    <template v-else-if="initializationState !== 'INITIALIZED'">
+    <template v-else-if="initializationState !== 'INITIALIZED' && initializationState !== 'MODIFIED' && initializationState !== 'NEW'">
```

Comment:
Add a clear computed to add a meaning to this condition.

But the problem here is that, the states `MODIFIED` and `NEW` are used when the user doesn't have really received the initial funds.